app/models/thredded/user_extender.rb

```diff
@@ -65,5 +65,11 @@ def thredded_anonymous?
     def following?(topic)
       UserTopicFollow.find_by(topic_id: topic.id, user: self)
     end
+
+    def thredded_display_name
+      send(Thredded.user_display_name_method).presence || fail(<<-ERROR)
```

Comment:
@glebm :-( I think rubocop thinks it's a perlism too:

```
 app/models/thredded/user_extender.rb:70:56: C: [Corrected] Style/AndOr: Use || instead of or.
      send(Thredded.user_display_name_method).presence or fail(<<-ERROR)
```